spyder/utils/programs.py

```diff
@@ -462,6 +462,24 @@ def is_module_installed(module_name, version=None, installed_version=None,
             
             return check_version(actver, version, symb)
 
+def is_python_interpreter(filename):
+    """Evaluate wether a file is a python interpreter or not."""
+    real_filename = os.path.realpath(filename) # To follow symlink if existent

```

Comment:
There should be 2 spaces after code finishes and a comment starts.

`real_filename = os.path.realpath(filename)  # To follow symlink if existent`